### PR TITLE
Fix 0.8.2

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: util/src/test/scala/org/alephium/util/AVectorSpec.scala [hunk 1]
```diff
@@ -385,6 +415,12 @@ abstract class AVectorSpec[@sp A: ClassTag](implicit ab: Arbitrary[A], cmp: Orde
       vc.indices.step is 1
     }
   }
+
+  it should "toIterable" in new Fixture {
+    forAll(vectorGen) { vc =>
+      vc.toIterable.toSeq is vc.toArray.toIterable.toSeq
```
Comment (tdroxler) [inline util/src/test/scala/org/alephium/util/AVectorSpec.scala:421]: can't you directly compare the `Iterable` rather doing the `toSeq`?

File: util/src/test/scala/org/alephium/util/ValueSortedMapSpec.scala [hunk 2]
```diff
@@ -51,4 +51,15 @@ class ValueSortedMapSpec extends AlephiumSpec {
...
+
+  it should "remove elements" in {
+    import akka.util.ByteString
+    import Bytes.byteStringOrdering
+    val map    = ValueSortedMap.empty[ByteString, TimeStamp]
+    val hashes = (0 until 100).map(_ => UnsecureRandom.nextU256().toBytes)
+    val now    = TimeStamp.now()
+    hashes.foreach(map.put(_, now))
+    map.remove(hashes.head)
+    map.remove(hashes.head)
```
Comment (tdroxler) [inline util/src/test/scala/org/alephium/util/ValueSortedMapSpec.scala:63]: I don't see any assertion, but I suppose previously removing twice the key was throwing an error

**Review by tdroxler — APPROVED** [review]
Very minor comment, LGTM

File: util/src/test/scala/org/alephium/util/ValueSortedMapSpec.scala (hunk 2 above)
Comment (polarker) [reply util/src/test/scala/org/alephium/util/ValueSortedMapSpec.scala:63]: Yes, the double removal does not work previously

File: util/src/test/scala/org/alephium/util/AVectorSpec.scala (hunk 1 above)
Comment (polarker) [reply util/src/test/scala/org/alephium/util/AVectorSpec.scala:421]: Unfortunately not. This might be a bug of Scala

**Pull request merged**